### PR TITLE
VIX-2615 Corrected coding error in identifying the active row

### PR DESCRIPTION
**Pull request by da-markson**

### CODE REVIEW

File: src/Vixen.Common/Controls/TimeLineControl/Grid_Mouse.cs [hunk 1]
```diff
@@ -967,7 +967,7 @@ private void MouseMove_HResizing(Point gridLocation, Point delta)
 					if (AltPressed && adjoiningElements.Count() == 0)
 					{
 						// Get the Elements that are preceding the Selected Element
-						List<Element> priorElements = Rows.First().GetPriorsOfElement(SelectedElements.First());
+						List<Element> priorElements = ActiveRow.GetPriorsOfElement(SelectedElements.First());
```
Comment (jeffu231) [inline src/Vixen.Common/Controls/TimeLineControl/Grid_Mouse.cs:970]: ActiveRow may not always be the same as the effect you are trying to adjust. Example. Drag a new effect to a row or an effect from any row to a new row. That new effect will be selected, and you can move / resize it, but the active row will not coincide. So probably the safest solution here is to capture SelectedElements.First() and then use the Row of that element which should be the row you want.

File: src/Vixen.Common/Controls/TimeLineControl/Grid_Mouse.cs [hunk 2]
```diff
@@ -1018,7 +1018,7 @@ private void MouseMove_HResizing(Point gridLocation, Point delta)
 					if (AltPressed && adjoiningElements.Count() == 0)
 					{
 						// Get the Elements that are following the Selected Element
-						List<Element> followElements = Rows.First().GetFollowersOfElement(SelectedElements.First());
+						List<Element> followElements = ActiveRow.GetFollowersOfElement(SelectedElements.First());
```
Comment (jeffu231) [inline src/Vixen.Common/Controls/TimeLineControl/Grid_Mouse.cs:1021]: Same as above.

File: src/Vixen.Common/Controls/TimeLineControl/Grid_Mouse.cs (hunk 1 above)
Comment (da-markson) [reply src/Vixen.Common/Controls/TimeLineControl/Grid_Mouse.cs]: Changing the asignment of ActiveRow may have unforseen ramifications. I instead changed to use your recommendation.

File: src/Vixen.Common/Controls/TimeLineControl/Grid_Mouse.cs (hunk 2 above)
Comment (da-markson) [reply src/Vixen.Common/Controls/TimeLineControl/Grid_Mouse.cs]: Changing the asignment of ActiveRow may have unforseen ramifications. I instead changed to use your recommendation.

**Review by jeffu231 — APPROVED** [review]

**Pull request merged**